src/python/pants/backend/project_info/list_roots_test.py

```diff
@@ -44,6 +44,7 @@ def test_multiple_source_roots(rule_runner: RuleRunner) -> None:
 
 
 def test_buildroot_is_source_root(rule_runner: RuleRunner) -> None:
+    rule_runner.create_dir("code")
     assert_roots(rule_runner, ["/"], expected=["."])
```

Comment:
`roots` only reports on used roots. Now that we ignore `.pants.d` in `RuleRunner`, there was no actual content in the build root, so `roots` was empty.